tools/proteore_venn_diagram/.shed.yml

```diff
@@ -0,0 +1,15 @@
+categories: [Proteomics]
+description: ProteoRE JVenn Diagram
+long_description: A tool for processing lists and produces Venn diagrams
+name: proteore_venn_diagram
+owner: proteore
```

Comment:
galaxyp

Reply:
right: the ownership "proteore" is going to be replaced by "galaxyp" in each ".yml" tools asap...